### ISSUE TITLE
Scrolling sideways using a mousewheel

### ISSUE THREAD

**Issue by IJsLauw** [davidfig/pixi-scrollbox#51]
How can I tell the scrollbar or rather the viewport to scroll on the x axis using the mousewheel?

From the api of viewport I thought this should have worked:

`scrollbox.content.wheel = false
  
scrollbox.content.drag( {
        direction: 'x',
        wheel: true
      })`

But it didn't, am I doing this all wrong? Can someone show me how to achieve this?

**Comment by davidfig** [comment]
I don't think there's support for that behavior in the code. After turning off the wheel listener in the pixi-scrollbox (or perhaps pixi-viewport?), you'll have to add your own wheel listener to support it. Just look for the wheel listener and then follow the code and you should find how we did the y scrolling.

**Comment by IJsLauw** [comment]
Ok, that sounds like that could work (too?), but what is above code supposed to do then?

![image](https://user-images.githubusercontent.com/6771143/191512882-4ea44869-e866-48dd-8643-b40ff552aeff.png)

Again looking at the above docs for pixi-viewport which (scrollbox.content is right?). To me it reads as I can use the _wheel_ to drag in the _direction_ supplied, if I disable the wheel plugin (for zooming)

**Comment by davidfig** [comment]
Yes, you can use drag just like it's used in pixi-scrollbox. But i don't think the drag plugin supports the wheel in the 'x' direction. You can check. If not, either create a new plugin for the scrollbox.content (viewport), or handle the wheel even in your client code.
